apps/vscode/plugins/plugins.py

```diff
@@ -0,0 +1,50 @@
+import json
+import pathlib
+
+from talon import Context, Module, actions
+
+mod = Module()
+ctx = Context()
+
+vscode_plugin_cache = None
+
+ctx.matches = r"""
+    app: vscode
+    tag: user.plugins
+"""
+
+
+def vscode_update_plugins_list():
+    """Queries the list of vscode plugins"""
+    extensions = pathlib.Path.home() / ".vscode/extensions/extensions.json"
+    if not extensions.exists():
+        return []
+    with extensions.open() as f:
+        return [x["identifier"]["id"] for x in json.loads(f.read())]
```

Comment:
```suggestion
        return [x["identifier"]["id"].lower() for x in json.loads(f.read())]
```

Just in case there are case differences, perhaps we should standardize on lower case?